Common/include/geometry/CGeometry.hpp

```diff
@@ -954,6 +954,19 @@ class CGeometry {
    */
   void SetCustomBoundary(CConfig* config);
 
+  /*!
+   * \brief General interface for setting the velocity in the geometry during motion ramps
+   * \param[in] config - config class
+   * \param[in] ramp_flag - flag for type of ramp
+   * \param[in] print - bool to print update to screen
+   */
+  void SetVelocity(CConfig* config, bool print) {
+    if (config->GetKind_GridMovement() == ENUM_GRIDMOVEMENT::ROTATING_FRAME)
+      SetRotationalVelocity(config, print);
+    else if (config->GetKind_GridMovement() == ENUM_GRIDMOVEMENT::STEADY_TRANSLATION)
+      SetTranslationalVelocity(config, print);
+  };
+
```

Comment:
used?

Reply:
No, this was still left from the merge of Josh' PR. I fixed it now